src/rust/engine/process_execution/src/remote.rs

```diff
@@ -1303,30 +1439,31 @@ pub async fn check_action_cache(
   metadata: &ProcessMetadata,
   platform: Platform,
   context: &Context,
-  action_cache_client: Arc<ActionCacheClient>,
-  headers: &BTreeMap<String, String>,
+  action_cache_client: Arc<ActionCacheClient<Channel>>,
+  _headers: &BTreeMap<String, String>,
```

Comment:
These used to be used? I guess they're not any more because they're applied in `new` instead? In which case, delete?

Reply:
Removed.